client/desktop/lib/debounce/index.js

```diff
@@ -0,0 +1,21 @@
+// Returns a function, that, as long as it continues to be invoked, will not
+// be triggered. The function will be called after it stops being called for
+// N milliseconds. If `immediate` is passed, trigger the function on the
+// leading edge, instead of the trailing.
+//
+// Ref: https://davidwalsh.name/javascript-debounce-function
+module.exports = function debounce( func, wait, immediate ) {
```

Comment:
Could you use the debounce function from lodash?

Reply:
👍 - see f2d3dc7a1d26380a5f43da45b7e9e83e16865419